x/blob/types/payforblob_test.go

```diff
@@ -174,9 +175,9 @@ func TestValidateBasic(t *testing.T) {
 			wantErr: nil,
 		},
 		{
-			name:    "parity shares namespace id",
-			msg:     paritySharesMsg,
-			wantErr: ErrParitySharesNamespace,
+			name:        "parity shares namespace id",
+			msg:         paritySharesMsg,
+			wantErrBool: true,
```

Comment:
quick question: do you mind pointing out the change that forces this? thanks!

Reply:
If the test case were instead:
```go
		{
			name: "parity shares namespace id",
			msg:  paritySharesMsg,
			// wantErrBool: true,
			wantErr: ErrParitySharesNamespace,
		},
```

we would encounter a test failure:

```
--- FAIL: TestValidateBasic (0.03s)
    --- FAIL: TestValidateBasic/parity_shares_namespace_id (0.00s)
        /Users/rootulp/git/rootulp/celestia/celestia-app/x/blob/types/payforblob_test.go:243: 
            	Error Trace:	/Users/rootulp/git/rootulp/celestia/celestia-app/x/blob/types/payforblob_test.go:243
            	Error:      	"unsupported namespace version 255" does not contain "cannot use parity shares namespace ID"
            	Test:       	TestValidateBasic/parity_shares_namespace_id
```

because this error is hit [here](https://github.com/rootulp/celestia-app/blob/c13409907a234fbd5e12231fb64edba140109af0/x/blob/types/payforblob.go#L105-L108) before the error [here](https://github.com/rootulp/celestia-app/blob/c13409907a234fbd5e12231fb64edba140109af0/x/blob/types/payforblob.go#L109-L112)

Reply:
certainly not blocking on this, but since this test only checks for inclusion of a specific error message, could we also either create an skderror for invalid namespace (would have the benefit of helping users see specific error codes), or change the test to accept `error` instead of `sdkerror.Error`, then use
```go
{
			name: "parity shares namespace id",
			msg:  paritySharesMsg,
			wantErr: errors.New("unsupported namespace version"),
},
```

Reply:
Good idea, resolved via https://github.com/celestiaorg/celestia-app/pull/1590/commits/93f74344b7394173d264151628b5fe076c57c9a0